platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/MapClickEvent.java

```diff
@@ -0,0 +1,171 @@
+package com.mapbox.mapboxsdk.module.telemetry;
+
+import android.annotation.SuppressLint;
+
+/**
+ * Event that represents users' gestures on map, for the details of gestures,
+ * please refer to {@link com.mapbox.mapboxsdk.constants.TelemetryConstants}
+ */
+@SuppressLint("ParcelCreator")
```

Comment:
with the base class already suppressing this lint error, I think we can remove this from all subclasses.

Reply:
If removed, it will still show lint error.